demo/index.html

```diff
@@ -61,14 +61,10 @@
       <h1>The raml-request-parameters-editor element</h1>
       <div main>
         <template is="dom-if" if="[[isMethod]]" restamp="true">
-          <raml-request-parameters-editor
-            query-parameters="[[selected.queryParameters]]"
-            uri-parameters="[[selected.allUriParameters]]"
-            url="[[selected.absoluteUri]]"
-            value="{{url}}"></raml-request-parameters-editor>
+          <raml-request-parameters-editor query-parameters="[[selected.queryParameters]]" uri-parameters="[[selected.allUriParameters]]" url="[[selected.absoluteUri]]" value="{{url}}"></raml-request-parameters-editor>
             <div class="url-value">
               <p>Produced URL value is:</p>
-              <anypoint-text-field value="{{url}}" label="Request URL"></anypoint-text-field>
+              <raml-request-url-editor auto-validate required uri-parameters="[[selected.allUriParameters]]" url="[[selected.absoluteUri]]" value="{{url}}" ></raml-request-url-editor>
```

Comment:
Not sure what you mean by that. But 1) it's still demo page 2) I ask for a review to check if the element changes anything visually (I added new element to a collection with new styles declaration). External API of the element haven't change.